ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/NetHelper.java

```diff
@@ -192,6 +193,22 @@ public static String getEnumValues(
                 .collect(Collectors.joining(",\n"));
     }
 
+    /**
+     * Creates a comma separated list of strings with the specified indentation.
+     * Used in Net model generation.
+     */
+    public static String commaSeparateStrings(
+            final ImmutableList<String> strings,
+            final int indent) {
+        if (isEmpty(strings)) {
+            return "";
+        }
+        return strings
+                .stream()
+                .map(i -> i)
+                .collect(Collectors.joining(",\n" + indent(indent)));
```

Comment:
Yes. This way I can have the function call indented in the template. It makes the template little bit more readable imo